tools/src/linter/SchemasValidator.ts

```diff
@@ -100,16 +92,10 @@ export default class SchemasValidator {
     }).filter(e => e != null) as ValidationError[]
   }
 
-  validate_schema (schema: Record<string, any>, key: string | undefined = undefined): Error | undefined {
+  validate_schema (schema: Record<string, any>): string | undefined {
     if (schema == null || schema.$ref != null) return
-    try {
-      if (key != null) this.ajv.addSchema(schema, key)
-      this.ajv.compile(schema)
-    } catch (_e: any) {
-      const error = _e as Error
-      for (const prefix of IGNORED_ERROR_PREFIXES) if (error.message.startsWith(prefix)) return
-      return error
-    }
+    const message = this.json_validator.validate_schema(schema)
+    if (message != null) return message
```

Comment:
Is this because of `null` vs. `undefined`? Should be able to just `return this.json_validator.validate_schema(schema)`.

Reply:
Nope just some brain fart when refactoring. Thanks for catching that.